stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.19
-version: 1.34.0
+version: 1.34.1
```

Comment:
`1.35.0` for new features. Other than that it looks good.
/assign

Reply:
bumped to 1.35.0 in https://github.com/helm/charts/pull/17987/commits/29b1dcbd2123dcb06df248c15024c9c4c0569e7b